android/fastlane/README.md

```
@@ -0,0 +1,48 @@
+fastlane documentation
+----
+
+# Installation
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)